chalicelib_smaht/checks/audit_checks.py

```diff
@@ -108,3 +108,36 @@ def check_submitted_md5(connection):
         check.summary = 'No inconsistent md5sums.'
         check.description = 'No inconsistent md5sums.'
     return check
+
+
+@check_function()
+def check_for_new_submissions(connection):
+    """ Weekly check that will compare against the previous week to determine if any new submissions
+        need attention
+    """
+    check = CheckResult(connection, 'check_for_new_submissions')
+    last_result = check.get_primary_result()
```

Comment:
Didn't know about this, but that's a neat feature. I assume it will also pull the last results if the check was run manually ran than on a schedule, but that should be fine, I think.

Reply:
Correct, you can do this from UI by specifying primary=true. You can also do ``get_latest_result`` if you don't want a "primary" one. By default "primary" results are what are displayed.